index.src.html

```diff
@@ -678,7 +687,14 @@
             {{VideoFramePlaneInit/cropHeight}}
             be the crop region of the decoded video frame |output| in
             pixels, prior to any aspect ratio adjustments.
-        5. Let {{VideoFramePlaneInit/displayWidth}} and
+        5. If |displayAspectWidth| and |displayAspectWidth| are provided,
+            assign the value of the decoded video frame in pixel to either
+            {{VideoFramePlaneInit/displayWidth}} or
+            {{VideoFramePlaneInit/displayHeight}}, and scale up the other
+            dimension to match the ratio of |displayAspectWidth| and
+            |displayAspectHeight|, as described in
+            [=intrinsic width|HTMLVideoElement=].
```

Comment:
Is there a better way to say this? Or at least to reference the spec?

Reply:
I would say "as described by [=intrinsic size=] in [[!HTML]]". But I would probably drop the reference. It describes the historical behavior for <video>, but it doesn't clearly point to a dfn for us to use in our algo steps.

My edit:

a. Let displayWidht and displayHeight be the display size of the decoded frame in pixels
b. if displayAspectWidth and displayAspectHeight are provided, increase displayWidth or displayHeight until the ratio of displayWidth : displayHeight equals that of displayAspectWidth : displayAspectHeight.
c. Assign displayWidth and displayHeight to {{VideoFramePlaneInit/displayWidth}} and {{VideoFramePlaneInit/displayHeight}} respectively.

Reply:
sorry, just edit'ed this comment - my first attempt trailed off because I confused the markdown editor w/ a video tag

Reply:
This is a nice clean way of saying this. TY!